.github/workflows/docker-publish-hf.yml

```diff
@@ -0,0 +1,86 @@
+name: Build and push HuggingFace TGI docker image
+
+on:
+  workflow_dispatch:
+    inputs:
+      tgi-version:
+        description: 'tgi version'
+        required: true
+        default: '0.5.0'
+
+jobs:
+  create-runner:
+    runs-on: [ self-hosted, scheduler ]
+    steps:
+      - name: Create new CPU instance
+        id: create_cpu
+        run: |
+          cd /home/ubuntu/djl_benchmark_script/scripts
+          token=$( curl -X POST -H "Authorization: token ${{ secrets.ACTION_RUNNER_PERSONAL_TOKEN }}" \
+          https://api.github.com/repos/deepjavalibrary/djl-serving/actions/runners/registration-token \
+          --fail \
+          | jq '.token' | tr -d '"' )
+          ./start_instance.sh action_cpu $token djl-serving
+    outputs:
+      cpu_instance_id: ${{ steps.create_cpu.outputs.action_cpu_instance_id }}
+
+  build-and-push-image:
+    runs-on: [ self-hosted, cpu ]
+    timeout-minutes: 150
+    needs: create-runner
+    env:
+      TGI_VERSION: ${{github.event.inputs.tgi-version}}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: huggingface/text-generation-inference
+          ref: v${{ env.TGI_VERSION }}
+      - uses: actions/checkout@v3
+        with:
+          repository: aws/deep-learning-containers
+          path: deep-learning-containers
+      - name: Setup Docker buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          install: true
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@v4
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
```

Comment:
why we still need this given credential inside?

Reply:
This is only setting up region. See https://github.com/aws-actions/configure-aws-credentials#self-hosted-runners.